### DISCUSSION TITLE
13.2 wont compile with SF memory cassette option

### DISCUSSION THREAD

**Discussion by handles888** [sanni/cartreader#953] [Coding Corner]
I dropped the Cart_reader 13.2 folder into the portable/sketch folder in a portable 13.1 environment.  Also replaced the sd folder.  Compiles just fine unless I try the SF cassette option which I do have and use.  Is it me doing something wrong or is something broken?  Works fine in 13.1 however I have Vselect and would like to use 13.2 because the voltage spike is fixed
![13 2](https://github.com/sanni/cartreader/assets/56324476/da8d7835-f317-4823-940a-0ea794541190)
![noSF](https://github.com/sanni/cartreader/assets/56324476/afe38095-485b-4d55-b7ba-4279b8917938)
![SFerror](https://github.com/sanni/cartreader/assets/56324476/d1fff4ef-6602-4f24-8a2a-44952179958f)

**Comment by PsyK0p4T** [comment]
Can you list all the modules you enabled in Config.h ? Only SFM or other ones too ?

**Reply by handles888** [reply]
I have enabled the following
V5
GB
SNES
NES/famicom
Genesis
N64
SMS
SF cassette
Vselect
Clockgen
3.3v fix
Self test
N64 fast CRC
Use updater thingy

Possibly the SNES flash repro but I need to double check.

That should be everything I have enabled.  Compiles fine in 13.1

**Reply by PsyK0p4T** [reply]
Try to enable these ones in Config.h too, compilation should be ok :
#define ENABLE_FLASH
#define ENABLE_FLASH16

**Reply by handles888** [reply]
Sorry I never tried this. I got really busy. I’m gonna check out 13.5 later today.

looks like something was updated in SFM.ino, perhaps I caught it while it was being edited
